pkg/goDB/protocols/protocols_generator.go

```diff
@@ -47,7 +53,10 @@ func readProtocols() (protoList []string, err error) {
 		}
 	}()
 
-	protoList = make([]string, 256)
+	// auxiliary to filter out duplicates
+	var seen = make(map[string]struct{})
```

Comment:
Why do we need duplicate filtering? The OS list should not have any (at least none that we don't want to actually track - and since it's a map it should overwrite anyway _if_ there are any duplicates from the key side)?

Reply:
Nvm, #234 ...